### PR TITLE
Update icu-globalization-api.md

### PR DESCRIPTION
**Pull request by danmoseley**
## Summary

At @PriyaPurkayastha suggestion, add a link from this breaking change doc to @safern's doc which provides more details about the different ways to switch between behaviors.

### CODE REVIEW

**Review by safern — APPROVED** [review]
Thanks, @danmosemsft

File: includes/core-changes/globalization/5.0/icu-globalization-api.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This change was introduced for two reasons:
 
 #### Recommended action
 
-No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior.
+No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior. The [.NET Globalization and ICU](https://docs.microsoft.com/dotnet/standard/globalization-localization/globalization-icu) document contains more information about the available switches.
```
Comment (mairaw) [inline includes/core-changes/globalization/5.0/icu-globalization-api.md:22]: If you want to use the standard verbiage

```suggestion
No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior. For more information about the available switches, see the [.NET Globalization and ICU](/dotnet/standard/globalization-localization/globalization-icu) article.
```

**Review by BillWagner — APPROVED** [review]
Thanks @danmosemsft 

This LGTM. I'll :shipit: now.

File: includes/core-changes/globalization/5.0/icu-globalization-api.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This change was introduced for two reasons:
 
 #### Recommended action
 
-No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior.
+No action is required on the part of the developer. However, if you wish to continue using NLS globalization APIs, you can set a [run-time switch](../../../../docs/core/run-time-config/globalization.md#nls) to revert to that behavior. For more information about the available switches, see the [.NET globalization and ICU](/dotnet/standard/globalization-localization/globalization-icu) article.
```
Comment (gewarren) [inline includes/core-changes/globalization/5.0/icu-globalization-api.md:22]: Isn't it better to use a relative link so it's validated at build time?

Comment (danmosemsft) [reply includes/core-changes/globalization/5.0/icu-globalization-api.md:22]: I didn't know that. Happy to create a new PR if you want this.

**Pull request merged**